sdk/helper/keysutil/policy.go

```diff
@@ -41,6 +42,8 @@ import (
 	"github.com/hashicorp/vault/sdk/helper/jsonutil"
 	"github.com/hashicorp/vault/sdk/helper/kdf"
 	"github.com/hashicorp/vault/sdk/logical"
+
+	"github.com/google/tink/go/kwp/subtle"
```

Comment:
this is a nice way to get forward secrecy, that's really cool

Reply:
`tink` isn't listed within the SDK's go.mod, which is technically should be if we are using it here?

Reply:
Isn't it?

Reply:
I think 812395cd3709326b1556c48753bf2300ddaed7c0 would've done so, but I might be wrong?